datalad/interface/common_opts.py

```diff
@@ -42,6 +43,36 @@
     constraints=EnsureInt() | EnsureNone(),
     doc="""limit recursion into subdataset to the given number of levels""")
 
+contains = Parameter(
+    args=('--contains',),
+    metavar='PATH',
+    action='append',
+    doc="""limit to the subdatasets containing the
+    given path. If a root path of a subdataset is given, the last
+    considered dataset will be the subdataset itself.[CMD:  This
+    option can be given multiple times CMD][PY:  Can be a list with
+    multiple paths PY], in which case datasets that
+    contain any of the given paths will be considered.""",
+    constraints=EnsureStr() | EnsureNone())
+
+fulfilled = Parameter(
+    args=("--fulfilled",),
+    doc="""DEPRECATED: use [CMD: --state CMD][PY: `state` PY]
+    instead. If given, must be a boolean flag indicating whether
+    to consider either only locally present or absent datasets.
+    By default all subdatasets are considered regardless of their
+    status.""",
+    constraints=EnsureBool() | EnsureNone())
```

Comment:
Does it make sense the centralize deprecated argument specifications? Doesn't it increase the locations that need adjustment?

Reply:
As you know I am all for centralization of all deprecation annotations!  Not sure though if it would be folded into this PR which is already sizeable and elderly.